Sources/Sentry/SentryAppStartTrackingIntegration.m

```diff
@@ -0,0 +1,71 @@
+#import "SentryAppStartTrackingIntegration.h"
+#import "SentryAppStartTracker.h"
+#import "SentryDefaultCurrentDateProvider.h"
+#import "SentryLog.h"
+#import <Foundation/Foundation.h>
+#import <SentryAppStateManager.h>
+#import <SentryClient+Private.h>
+#import <SentryCrashAdapter.h>
+#import <SentryDispatchQueueWrapper.h>
+#import <SentryHub.h>
+#import <SentrySDK+Private.h>
+#import <SentrySysctl.h>
+
+@interface
+SentryAppStartTrackingIntegration ()
+
+#if SENTRY_HAS_UIKIT
+@property (nonatomic, strong) SentryAppStartTracker *tracker;
+#endif
+
+@end
+
+@implementation SentryAppStartTrackingIntegration
+
+- (void)installWithOptions:(SentryOptions *)options
+{
+#if SENTRY_HAS_UIKIT
+    if (options.enableAppStartMeasuring && options.tracesSampleRate != nil &&
+        [options.tracesSampleRate doubleValue] != 0.0) {
```

Comment:
H: still missing if the sampler callback is not null, see Java https://github.com/getsentry/sentry-java/blob/b45944c93d53f68bc1e64f5ac4c9a5400b252d44/sentry/src/main/java/io/sentry/SentryOptions.java#L1318-L1326

Reply:
I don't understand why I need a `tracesSampler`? If you don't set one and only set a `tracesSampleRate`, the `SentryTracesSampler` uses the `tracesSampleRate`. The sampler callback can be different for every call. I don't think we should check for that here. Am I missing something?

Reply:
I fixed it now.